spec/models/budget_spec.rb

```diff
@@ -113,10 +115,16 @@
   context "when the period start and period end dates are invalid" do
     it "performs budget date validation" do
       budget = build(:budget, period_start_date: Date.today, period_end_date: Date.today + 366.days)
+  it "returns an instance of FinancialYear for the financial year" do
```

Comment:
Might be worth using `travel_to` and locking this into an exact date and known values for the expectations?